README.md

```diff
@@ -1,2 +1,83 @@
+
+## 📝 Description
+
+    reps.mozilla.org data on mobile
+
+## 🚀 Quick start
+
+1.  **Install Flutter on your machine**
+
+    install flutter by selecting the operating system on which you are installing Flutter:
+  
+    [Flutter installation tutorial](https://flutter.dev/docs/get-started/install)
+
+2.  **Update your path**
+
+    If you wish to run Flutter commands in the  console, take these steps to add Flutter to the PATH environment variable:
+    
+    [Windows](https://flutter.dev/docs/get-started/install/windows)
+    
+    [MacOS](https://flutter.dev/docs/get-started/install/macos)
+    
+    [Linux](https://flutter.dev/docs/get-started/install/linux)
+
+
+    
+
+3.  **Run flutter doctor**
+
+    This command checks your environment and displays a report of the status of your Flutter installation. Check the output carefully for other software you might need to install or further tasks to perform.
+    
+    Windows: https://flutter.dev/docs/get-started/install/windows#update-your-path
+    
+    MacOS:   https://flutter.dev/docs/get-started/install/macos#update-your-path
+    
+    Linux:   https://flutter.dev/docs/get-started/install/linux#update-your-path
+    
+    
+4.  **Platform setup**
+  
+    Flutter relies on a full installation of Android Studio to supply its Android platform dependencies. However, you can write your Flutter apps in a number of editors
+    
+    Windows: https://flutter.dev/docs/get-started/install/windows#android-setup
+    
+    MacOS:    https://flutter.dev/docs/get-started/install/macos#platform-setup
+    
+    Linux:     https://flutter.dev/docs/get-started/install/linux#android-setup
+    
+5.  **Set up an editor**
+  
+    You can build apps with Flutter using any text editor combined with flutter command-line tools. However, we recommend using one of flutter editor plugins for an even better experience. These plugins provide you with code completion, syntax highlighting, widget editing assists, run & debug support, and more.
+    
+    Windows:   https://flutter.dev/docs/get-started/editor?tab=androidstudio
+    
+    MacOS:   https://flutter.dev/docs/get-started/editor?tab=androidstudio
+    
+    Linux:     https://flutter.dev/docs/get-started/editor?tab=androidstudio
+    
+    
+6.  **Test drive**
+    This  describes how to create a new Flutter app from templates, run it, and experience “hot reload” after you make changes to the app.
+    
+     Windows:   https://flutter.dev/docs/get-started/test-drive?tab=androidstudio
+    
+    MacOS:  https://flutter.dev/docs/get-started/test-drive?tab=androidstudio
+    
+    Linux:     https://flutter.dev/docs/get-started/test-drive?tab=androidstudio
+    
+7.  **Learn more!**
+
+    Learn more about the Flutter framework from  [here](https://flutter.dev/docs/get-started/learn-more).
```

Comment:
This whole document feels like a Flutter guide rather than a guide for the actual app. If you're adding installation steps, please add steps on the installation of the app on a local environment and not steps to install Flutter. The Readme should be an insight into the app and what is it about along with some steps to install it so that new contributors understand what the repository is about.
Please consider the above and make suitable changes.